src/components/common/modal/ModalWrapper.js

```diff
@@ -64,6 +64,9 @@ const getStylesFromProps = ({ theme }) => ({
   noneShadow: {
     boxShadow: 'none',
   },
+  maxHeightBlock: {
+    maxHeight: theme.sizes.maxHeightForTabletAndDesktop,
```

Comment:
this is for tablet and desktop but there is not condition to check its not on mobile

Reply:
why does the pop take the whole height? it is the same with other feed popups?

Reply:
These are the height limits of the main pop-up block for all devices. Since on some mobile devices, this does not look good. This is visible on the screenshots.

Reply:
[done](https://github.com/GoodDollar/GoodDAPP/pull/678/files/fc487352cd3e7d4feaf6fbcebce77005ed9914e6#diff-2f7b6754873b5f64d1921b8da7b21143R28)